index.html

```diff
@@ -1501,6 +1501,53 @@ <h4>LinkedDomains</h4>
 
   </section>
 
+  <section>
+    <h1>Representations</h1>
+    <p>To reserve something at IANA in the "standards tree", it needs to exist in specification form. </p>
+    <p>This table allows for registration of new representation specifications.</p>
+    <table class="simple" style="width: 100%;">
+      <thead>
+        <tr>
+          <th>Media Type</th>
+          <th>Specification</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr id="application-did-json">
+          <td>application/did+json</td>
+          <td>
+            <a href="https://www.w3.org/TR/did-core/#application-did-json">DID Core</a>
+          </td>
+        </tr>
+        <tr id="application-did-ld-json">
+          <td>application/did+ld+json</td>
+          <td>
+            <a href="https://www.w3.org/TR/did-core/#application-did-ld-json">DID Core</a>
+          </td>
+        </tr>
+        <tr id="application-did-cbor">
+          <td>application/did+cbor</td>
+          <td>
+            <a href="https://www.w3.org/TR/did-core/#application-did-cbor">DID Core</a>
+          </td>
+        </tr>
+        <tr id="application-did-dag-cbor">
+          <td>application/did+dag+cbor</td>
+          <td>
+            <p class="issue" data-number="252"></p>
+          </td>
+        </tr>
+        <tr id="application-did-yaml">
+          <td>application/did+yaml</td>
+          <td>
+            <p class="issue" data-number="253"></p>
+          </td>
+        </tr>
```

Comment:
```suggestion
```

We shouldn't register things that don't exist yet.

Reply:
https://github.com/transmute-industries/did-core/tree/main/packages/did-dag-cbor
https://github.com/transmute-industries/did-core/tree/main/packages/did-yaml

The specifications may not exist yet, but the representations do.... I am happy to write the specs before we merge the PR, but I thought it might bee better to try and gather input on issues / give the community more time to realize why they need to work together to register these things.

Reply:
> The specifications may not exist yet, but the representations do.

I'm not going to block this -- just raise my deep concern about representation proliferation.

The ideal is that we have one representation, it focuses effort and stops us getting mired in representation-specific discussions (for representations that most everyone is not going to use). Like... do we really need a did+yaml representation? I'd argue that we don't... and by having one, we harm interop.

That said, I don't want to block folks that really, really want a did+yaml representation.